### ISSUE TITLE
[BUG] Steam deck stream inputs not recognizing 

### ISSUE THREAD

**Issue by NarrowMacaroon** [streetpea/chiaki-ng#396]
i got chiaki NG to work on my steam deck but i have an issue with the inputs, essentially sometimes my inputs go through and sometimes they dont, i have to spam a button for it to go through, steam deck is latest version, stable and beta have been tested both dont seem to do anything, tried in both desktop and game mode and both do the same thing, a little help would really be appreciated

**Comment by streetpea** [comment]
Please make sure you are using the default control scheme for chiaki-ng. You can get it by searching for chiaki4deck+ mic by gmoney23 in the community layouts tab or adding the steam game via the create steam shortcut button.

**Comment by NarrowMacaroon** [comment]
I have the control scheme from gmoney, and i havent messed with the control scheme for chiaki

**Comment by streetpea** [comment]
Is your connection very choppy as well. It is possible you're controller input is not going through to your PlayStation (i.e., the packet is dropped) due to extremely poor connection quality. However, it's more likely an issue with SteamOS as chiaki-ng just uses SDL to pick up inputs and works fine on my machine with the latest stable version.

**Comment by NarrowMacaroon** [comment]
yes ive tried to hook up my ps5 controller to my steam deck and the issue persists, the stream is fine and its running on a local network, as i said i tried both stable and pre release versions of steam os and both seem to not do any difference for me

**Comment by streetpea** [comment]
Have you tried multiple games?

**Comment by NarrowMacaroon** [comment]
yes even the ps5 ui freaks out, i tried both local and over the internet and both just do the same thing which is have major lag on inputs or not input registration at all

**Comment by streetpea** [comment]
Please post a log with verbose logging enabled

**Comment by NarrowMacaroon** [comment]
here is the log https://pastebin.com/QipwFxab , idk how to turn on verbose logging

**Comment by NarrowMacaroon** [comment]
here is the new log with verbose on https://drive.google.com/file/d/1F_X0u026To3Pu5SOpeaBZxF1M9lNGXMI/view?usp=drive_link, paste bin wouldnt let me post it since its larger than 512 KB

**Comment by streetpea** [comment]
@NarrowMacaroon it seems chiaki-ng is streaming your inputs fine. Thus, it's likely a problem registering inputs / something running on your Steam Deck interfering with the inputs.

**Comment by streetpea** [comment]
@NarrowMacaroon have you tried attaching an external controller?

**Comment by NarrowMacaroon** [comment]
Yeah i have both an Xbox and ps5 controller both tried to use over Bluetooth and got nothing to work same issues with inputs not being registered

**Comment by NarrowMacaroon** [comment]
The only mods I’ve done to my steam deck is deckyloader and emu deck

**Comment by streetpea** [comment]
This build will log button presses so you can see if they are registering if you launch it via the konsole in desktop mode and look at the console as you input buttons. This will help determine if the issue is actually in registering the inputs or packets getting dropped by the PlayStation https://github.com/streetpea/chiaki-ng/actions/runs/10981768188/artifacts/1963288164
